dspace-api/src/main/java/org/dspace/submit/model/UploadConfiguration.java

```diff
@@ -53,7 +53,9 @@ public void setMaxSize(Long maxSize) {
 
     public Boolean isRequired() {
         if (required == null) {
-            required = configurationService.getBooleanProperty("webui.submit.upload.required");
+            //defaults to true
+            //don't store a local copy of the configuration property
+            return configurationService.getBooleanProperty("webui.submit.upload.required", true);
```

Comment:
@benbosman : I'm curious why we are *not* storing this configuration in the `required` local param?  This seems to be different than how we treat other fields in this `UploadConfiguration`, e.g. see the `getMaxSize()` method: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/submit/model/UploadConfiguration.java#L43

Are you implying that we need to change the behavior here for all the fields?  Or, is this field somehow unique?

Reply:
if we store it in a local param, we can't override it in ITs and test both settings

Reply:
@benbosman: Ugh, that implies to me that `UploadConfiguration` itself likely needs rework (or even possibly removal).  You are correct obviously that ITs/tests need to be able to override configuration easily....if `UploadConfiguration` causes that to not be possible, it may need to go away. 

(To clarify, I'm not saying that's required from this PR, it could happen in a follow-up or later PR.  It just sounds to me like this `UploadConfiguration` class is not serving any clear purpose.)